app/views/articles/_algolia_search.html.erb

```diff
@@ -83,8 +84,14 @@
       exactOnSingleWordQuery: "none",
       attributesToSnippet: ['body_text:19', 'comments_blob:8'],
     }
+
     if (filters === "MY_POSTS" && userData()) {
       searchObj["tagFilters"] = [['user_' + userData()['id']]]
+    } else if (hashtags != null && hashtags.length > 0) {
+      for(i = 0; i < hashtags.length; i++){
```

Comment:
I think you're missing a `var` here: `for (var i = 0...`

Reply:
Thank you, you are correct. I have pushed a new commit to remedy this oversight.